src/oc/web/components/org_dashboard.cljs

```diff
@@ -171,7 +173,11 @@
                                     (s/starts-with? (name open-panel) "follow-ups-picker-"))
         show-mobile-cmail? (and cmail-state
                                 (not (:collapsed cmail-state))
-                                is-mobile?)]
+                                is-mobile?)
+        user-responded-to-push-permission? (drv/react s :user-responded-to-push-permission?)
+        show-push-notification-permissions-modal? (and is-mobile?
+                                                       (jwt/jwt)
```

Comment:
@cjsauer just FYI, i added the jwt check to make sure we show the push notifications wall only to logged in users.
Since a user could be visiting a public org anonymously.
I initially used `user-is-part-of-the-team?` check but i think we can add the push token even if you are looking to a public org, we just need your user to set the push token to. Then you'll receive push only for your orgs.
Does it make sense to you?